src/components/fileDownloadCard.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect, useState } from "react";
+
+import {
+    Box,
+    HStack,
+    Icon,
+    Link,
+    Text,
+    VStack,
+    Menu,
+    MenuList,
+    MenuButton,
+    MenuDivider,
+    MenuItem,
+} from "@chakra-ui/react";
+import { MdDescription } from "react-icons/md";
+import colourTheme from "@styles/colours";
+import { updateFileApproval } from "@utils/updateFileApproval";
+
+type FileDownloadCardProps = {
+    filePath: string;
+    docLink: string;
+    docName: string;
+    docApproved: boolean | null;
+    participantEmail: string;
+};
+
+const FileDownloadCard: React.FC<FileDownloadCardProps> = ({
+    filePath,
+    docLink,
+    docName,
+    docApproved,
+    participantEmail,
+}): JSX.Element => {
+    const [approvalState, setApprovalState] = useState<boolean | null>(
+        docApproved,
+    );
+
+    useEffect(() => {
+        console.log(filePath, participantEmail, approvalState);
+        updateFileApproval(filePath, participantEmail, approvalState);
+    }, [approvalState, participantEmail]);
+
+    return (
+        <Box
+            borderWidth="1px"
+            borderColor="#C1C1C1"
+            width="full"
+            display="flex"
+            justifyContent="flex-start"
+            alignItems="flex-start"
+            height="200px"
+        >
+            <HStack w="full" h="full" alignItems="flex-start" spacing="10px">
+                <Icon
+                    as={MdDescription}
+                    color={colourTheme.colors.Blue}
+                    w={110}
+                    h={110}
+                    marginLeft="40px"
+                    marginTop="40px"
+                />
+                <VStack alignItems="flex-start" w="300px" h="full">
+                    <Text fontSize="18px" marginTop="75px" as="u">
+                        <Link href={docLink} color={colourTheme.colors.Blue}>
```

Comment:
I think we need something like an isExternal prop for this to open in another tab?
```suggestion
                        <Link href={docLink} color={colourTheme.colors.Blue} isExternal>
```